website/content/api-docs/secret/openldap.mdx

```diff
@@ -307,6 +307,10 @@ strings ("1h") or an integer number of seconds. Defaults to system/mount default
 be less than the mount max TTL (or, if not set, the system max TTL), but it is not allowed to be longer. The
 [available units](https://golang.org/pkg/time/#ParseDuration) are: `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
 
+The `creation_ldif`, `deletion_ldif`, `rollback_ldif`, and `username_template` fields are all templated fields. See
+[Username Templating](/docs/concepts/username-templating) for details on how to use templating. Also see
+[#Templates](Templates) for specifics on what data is available for each template.
```

Comment:
`[#Templates](Templates)` dead link?  (from the PR preview, the resulting link is https://vault-mot4pi4um.vercel.app/api-docs/secret/Templates)

Reply:
Whoops - inverted link format `[Templates](#Templates)`

Reply:
👍 unfortunately that still doesn't go anywhere for me (https://vault-1qjyd1t1v-hashicorp.vercel.app/api-docs/secret/openldap#Templates)

Reply:
Fixed in 3fdec70 and da4182a